spec/rubocop/cop/rspec/multiple_describes_spec.rb

```diff
@@ -50,4 +50,139 @@
       end
     RUBY
   end
+
+  context 'when EnforcedStyle is :splitting' do
+    let(:cop_config) { { 'EnforcedStyle' => :splitting } }
+
+    it 'check first describe in nested describe' do
+      expect_offense(<<-RUBY)
+        RSpec.describe MyClass do
+          describe '#do_something' do
+          ^^^^^^^^^^^^^^^^^^^^^^^^ Do not use nested describe - try to split them.
+            subject { my_class_instance.foo }
+          end
+        end
+      RUBY
+    end
+
+    it 'check first describe in each nested describe' do
+      expect_offense(<<-RUBY)
+        describe MyClassController, type: :controller do
+          describe 'do_something' do
+          ^^^^^^^^^^^^^^^^^^^^^^^ Do not use nested describe - try to split them.
+            describe 'do_something_in_describe' do; end
+            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Do not use nested describe - try to split them.
```

Comment:
But how? What is the suggested way to write this test?

Reply:
I created a new test to check the nested example groups only.